### PR TITLE
dbadvisor-17 Automated deployment to maven repo

### PR DESCRIPTION
**Pull request by mbocek**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +26,10 @@ before_install:
 script:
 - ./mvnw clean install
 
+after_success:
+- 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_9047eb4b7969_key -iv $encrypted_9047eb4b7969_iv -in cd/codesigning.asc.enc -out cd/codesigning.asc -d; gpg --fast-import cd/codesigning.asc; fi'
+- 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then mvn deploy -Prelease --settings cd/settings.xml; fi'
```
Comment (thradec) [inline .travis.yml]: isn't it strange to use release profile when we are not releasing real version?

Comment (thradec) [inline .travis.yml]: shouldn't we add skipTests so they won't run again?

Comment (mbocek) [reply .travis.yml]: Actually, we are releasing but the snapshot. The mechanism is same for releasing to (SNAPSHOT/Release) - only target repository is different.

Comment (mbocek) [reply .travis.yml]: fixed

File: pom.xml [hunk 2]
```diff
@@ -168,4 +168,76 @@
         </repository>
     </repositories>
 
+    <profiles>
```
Comment (thradec) [inline pom.xml]: I saw that some projects use org.sonatype.oss:oss-parent, which simplifies the configuration, WDYT?

Comment (mbocek) [reply pom.xml]: Check the URL: https://github.com/sonatype/oss-parents

Comment (thradec) [reply pom.xml]: oops, pity

**Pull request merged**